packages/terra-application-links/src/ApplicationLinks.js

```diff
@@ -1,10 +1,25 @@
+import PropTypes from 'prop-types';
 import ApplicationTabs from './tabs/ApplicationTabs';
-import ApplicationList from './list/ApplicationList';
+
+const ApplicationLinksPropType = PropTypes.arrayOf(PropTypes.shape({
+  /**
+   * The id to append to the link.
+   */
+  id: PropTypes.string,
+  /**
+   * The path to push to the route.
+   */
+  path: PropTypes.string.isRequired,
+  /**
+   * The display text for the link.
+   */
+  text: PropTypes.string.isRequired,
+}));
 
 const ApplicationLinks = {
   ApplicationTabs,
-  ApplicationList,
+  ApplicationLinksPropType,
 };
 
 export default ApplicationLinks;
-export { ApplicationTabs, ApplicationList };
+export { ApplicationTabs, ApplicationLinksPropType };
```

Comment:
Why are we exporting these proptypes?

Reply:
We are exporting them because header and application will both use them as part of their api.